test/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildAnAppWithLibrary.cs

```diff
@@ -201,5 +201,21 @@ public void The_clean_target_removes_all_files_from_the_output_folder()
             outputDirectory.Should().OnlyHaveFiles(Array.Empty<string>());
 
         }
+
+        [Fact]
+        public void An_appx_app_can_reference_a_cross_targeted_library()
+        {
+            var asset = _testAssetsManager
+                .CopyTestAsset("AppxReferencingCrossTargeting")
+                .WithSource()
+                .Restore("Appx");
+
+            var buildCommand = new BuildCommand(Stage0MSBuild, Path.Combine(asset.TestRoot, "Appx"));
+
+            buildCommand
+                .Execute()
+                .Should()
+                .Pass();
```

Comment:
Can you confirm that this test fails without the change?  The specified symptoms are an error but a passing build - is this verification sufficient for that case?